cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -244,17 +244,33 @@ public void upgrade(String tableName, Properties properties) throws Exception {
     }
   }
 
-  private void drop(byte[] tableName) throws IOException {
+  private void drop(TableId tableId) throws IOException {
     HBaseAdmin admin = getHBaseAdmin();
-    if (admin.tableExists(tableName)) {
-      admin.disableTable(tableName);
-      admin.deleteTable(tableName);
+    if (tableUtil.tableExists(admin, tableId)) {
+      tableUtil.disableTable(admin, tableId);
+      tableUtil.deleteTable(admin, tableId);
     }
   }
 
+  public TableId getConfigTableId(QueueName queueName) {
+    return getConfigTableId(queueName.getFirstComponent());
+  }
+
+  public TableId getConfigTableId(String namespace) {
+    return TableId.from(getConfigTableName(namespace));
+  }
+
+  public TableId getDataTableId(QueueName queueName) {
+    return TableId.from(getActualTableName(queueName));
+  }
+
+  public TableId getTableIdFromFlow(String namespaceId, String app, String flow) {
+    return TableId.from(getTableNameForFlow(namespaceId, app, flow));
+  }
+
   private void deleteConsumerConfigurations(QueueName queueName) throws IOException {
     // we need to delete the row for this queue name from the config table
-    HTable hTable = new HTable(getHBaseAdmin().getConfiguration(), getConfigTableName(queueName));
+    HTable hTable = tableUtil.getHTable(getHBaseAdmin().getConfiguration(), getConfigTableId(queueName));
```

Comment:
In general, if you are replacing a "new XXX" to a method on util class, it should be "createXXX".